lerobot/scripts/control_robot.py

```diff
@@ -4,6 +4,9 @@
 - Unlimited teleoperation at highest frequency (~200 Hz is expected), to exit with CTRL+C:
 ```bash
 python lerobot/scripts/control_robot.py teleoperate
+
+# Remove the cameras from the robot definition. They are not used in 'teleoperate' anyway.
+python lerobot/scripts/control_robot.py teleoperate --robot-overrides '~cameras'
```

Comment:
I'm not very fond of this `'~cameras'` syntax.
Why not have a `cameras.enable=true` for consistency with the rest of the configs?

Reply:
I will remove '~cameras' in the documentation.
I would prefer if people dont have cameras, they could remove the cameras directly in the config yaml.
I dont what to add enable=true here, I dont think it makes sense for defining robots.

Reply:
Changed my mind about removing  '~cameras'